ci/docker/manylinux.Dockerfile

```diff
@@ -7,6 +7,7 @@ ARG BUILDKITE_BAZEL_CACHE_URL
 ENV BUILD_JAR=1
 ENV RAY_INSTALL_JAVA=1
 ENV BUILDKITE_BAZEL_CACHE_URL=$BUILDKITE_BAZEL_CACHE_URL
+ENV PATH=/opt/python/cp38-cp38/bin:$PATH
```

Comment:
move it here so python is available globally in this image

Reply:
the python that it uses is different for each version.

this is only the path for py38.

should we worry that a wrong python version is being used down the line somewhere?

Reply:
I feel that the copy and upload wheel part should be executed in forge, rather than inside manylinux.

Reply:
good point, let me move it to forge